helpers/filterSeats.js

```diff
@@ -60,19 +69,73 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+    }
+
+    getBestSections();
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      // ticketRateSection
+      var ticketRateId = ticketRate.ids;
```

Comment:
Interesting this is making `ticketRateId` (singular) from `ticketRate.ids` (plural)... is it possible this could contain more than one id later? Perhaps if not, `ids` should be `id` from the API reply (raise JIRA to fix if so).